### ISSUE TITLE
Should this work with rangeslider? 

### ISSUE THREAD

**Issue by johann-petrak** [predict-idlab/plotly-resampler#156]
In order to make it easier to scroll through a huge dataset, I have added a rangeslider like so

```
fig.update_layout(
    xaxis=dict(
        autorange=True,
        rangeslider=dict(
            autorange=True,
            visible=True,
        ),
        type="date"
    ),
    yaxis=dict(
   ...
```

This does indeed show a functioning rangeslider, but unlike with pure plotly, nothing is shown within the range slider (normally the whole graph is outlined in the slider)

It also seems that there is no dynamic resampling going on when using the slider to change the shown window but this is hard to figure out exactly. 

UPDATE: actually, i was initially using fig.show() but with display(fig), the behavior with the rangeslider is even weirder as the slider window seems to somehow adapt to the resampling process. Still no data shown in the slider though. 

Is this supposed to work properly with resampling and show the graph outline in the slider?

If not, would there be an alternate way to get a "scroll bar" to quickly move within the graph? 

As I understand, without the rangeslider, the only way to scroll is by first zooming in and then "dragging" with the mouse at the bottom of the graph, but this is extremely slow and inefficient for large graphs to get around quickly. Constantly zooming fully out and back in is also not a good solution.

**Comment by jonasvdd** [comment]
Hi @johann-petrak,

I totally agree with you; a [range-slider](https://plotly.com/python/range-slider/) is useful when navigating oneself in a *large* time series. 

However, as plotly-resampler performs dynamic updates of trace data, the rangeslider will be updated as well when performing graph range interactions. As such, it will require some additional functionality (or a small hack somewhere), to make this work.

We have not yet found the time to look into this, (but we will add it to our long list of todo's)

And ofcourse, you are more than welcome to contribute with this matter!

Kind regards,
Jonas

**Comment by johann-petrak** [comment]
Thanks - I found plotly-resampler while looking for something that is perhaps even less complex: just a simple way to scroll a window (which might need to get defined initially) through a much wider plot using a scrollbar. 

So if something like this could get added/implemented easier than fixing the rangeslider approach, it would be useful as well! Perhaps it would even be possible to pull this off with jupyter widgets and any plotting library underneath, but I was hoping somebody would have invented that wheel already! 

I was unable (so far) to find any good solution in python, using any of the other plot libraries which would support easy horizontal scrolling of an initial window through a large time-series/horizontal scatter/line plot.

**Comment by jonasvdd** [comment]
fyi: in #254, we provide rangeslider-alike support!

**Comment by jonasvdd** [comment]
Rangeslider-alike functionality is available from `plotly-resampler>=0.9.2rc2`.

Make sure to look at the [docs](https://predict-idlab.github.io/plotly-resampler/latest/getting_started/#overview) and [examples](https://github.com/predict-idlab/plotly-resampler/tree/main/examples)!

Kind regards,
Jonas
